src/components/organisms/Hero.jsx

```diff
@@ -80,9 +80,10 @@ const HeroTagline = styled(P)`
   font-size: 1.15em;
   font-style: italic;
   font-weight: 100;
-  margin-top: 0;
-  margin-bottom: 0.5em;
+  margin: 0 auto 0.5em;
```

Comment:
🤔 It looks like something similar to `2em` feels more comfortable for top/bottom margin. But maybe others have thoughts on the aesthetic side of things.

```suggestion
  margin: 2em auto;
```

Reply:
Looks good to me, I'll make the change unless anyone objects.